pkl-gradle/src/test/kotlin/org/pkl/gradle/KotlinCodeGeneratorsTest.kt

```diff
@@ -125,6 +140,7 @@ class KotlinCodeGeneratorsTest : AbstractTest() {
             sourceModules = ["mod.pkl"]
             outputDir = file("build/generated")
             settingsModule = "pkl:settings"
+            ${if (kotlinPackage != null) "kotlinPackage = \"$kotlinPackage\"" else ""}
```

Comment:
and you used the more kotlinesque way above ;)
```suggestion
            ${kotlinPackage?.let { "kotlinPackage = \"$it\"" } ?: ""}
```